include/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(LIB_NAME yui-devel)
+
+add_library(${LIB_NAME} INTERFACE)
+
+target_link_libraries(${LIB_NAME} INTERFACE project_warnings)
```

Comment:
`code_analysis` should link `project_warnings`

Reply:
Agreed.